apps/judicial-system/backend/src/app/modules/case-table/caseTable.cellGenerators.ts

```diff
@@ -247,36 +247,43 @@ const generateRequestCaseStateTag = (
 }
 
 const generateIndictmentRulingDecisionTag = (
-  indictmentRulingDecision?: CaseIndictmentRulingDecision | null,
-): CaseTableCell<TagValue> => {
-  switch (indictmentRulingDecision) {
+  c: Case,
+  user: TUser,
+): CaseTableCell<TagValue | TagPairValue> => {
+  const createCell = (tag: TagValue, sortValue: string) =>
```

Comment:
We add a "correcting" tag to indictments that are being corrected, but only for district court users. Other users see these cases as normal completed cases.